ui-modules/utils/server-status/server-status.js

```diff
@@ -65,6 +65,9 @@ export function BrServerStatusDirective() {
                 }else if(response.status === 404) {
                     state = BrServerStatusModalController.STATES.NO_CONNECTION;
                 }else if(response.status === 401 || response.status === 403 ) {
+                    if( response.headers("LOGIN_PAGE")) {
```

Comment:
Is it really bad practice to invent response headers?

Reply:
Frankly, I do not know this bit, look at the `$location` service docs https://docs.angularjs.org/guide/$location

Reply:
I don't think it is. But by convention, there should start with `X_`. 

For instance, I think `X_BROOKLYN_LOGIN_PAGE` would be nicer